src/v/kafka/server/usage_manager.cc

```diff
@@ -299,6 +298,18 @@ ss::future<> usage_manager::accounting_fiber::async_data_fetch(
     if (!is_bucket_stale()) {
         _buckets[index].u = kafka_stats;
     }
+
+    /// Grab cloud storage stats via health monitor
+    const auto expiry = std::min<std::chrono::seconds>(
+      (_usage_window_width_interval * _usage_num_windows),
+      std::chrono::seconds(10));
+    co_await _health_monitor.maybe_refresh_cloud_health_stats();
```

Comment:
i recall that this method was checking if the current node was the raft0 leader. does this also need to be run on a specific core?

Reply:
I don't think so because the health monitor frontend dispatches all calls to the hm backend shard, but in either case the accounting_fiber is allocated, and started on shard-0

Reply:
its hard to judge in isolation. is it triggered from an admin_server? normally you'd redirect to the location where the operation isn't a noop.

Reply:
This portion of code is triggered via the window close timer